_layouts/post.html

```diff
@@ -34,11 +34,20 @@ <h1 class="post-title text-center">{{ page.title }}</h1>
       {% if page.meta %}
         &middot; {{ page.meta }}
       {% endif %}
-    </p>
+      {% if page.tags %}
```

Comment:
I would place `if tags` inside the `tags` partial instead of repeating it everywhere.